LICENSE.txt

```diff
@@ -0,0 +1,4 @@
+Picture of the factory (job-dsl-plugin/src/main/webapp/img/factory.png) was downloaded from
```

Comment:
I think It should contain plugin license as first entry and then mention about picture license